test/ciflow-bot.test.ts

```diff
@@ -187,6 +188,44 @@ describe('CIFlowBot Integration Tests', () => {
     scope.done();
   });
 
+  test('pull_request.opened event: add_default_labels strategy for a_random_user', async () => {
+    const event = require('./fixtures/pull_request.opened.json');
+    event.payload.pull_request.number = pr_number;
+    event.payload.pull_request.user.login = 'a_random_user';
+    event.payload.repository.owner.login = owner;
+    event.payload.repository.name = repo;
+
+    const scope = nock('https://api.github.com')
+      .post(`/repos/${owner}/${repo}/issues/${pr_number}/labels`, body => {
+        expect(body).toMatchObject(['ciflow/default']);
+        return true;
+      })
+      .reply(200);
+
+    await p.receive(event);
+
+    if (!scope.isDone()) {
+      console.error('pending mocks: %j', scope.pendingMocks());
+    }
+    scope.done();
+  });
+
+  test('pull_request.opened event: respect opt-out users', async () => {
```

Comment:
i'm not too familiar with testing in TypeScript but what does this actually check given that there's no `expect` for any results

Reply:
it basically make sure there's no extra api sent to `const scope = nock('https://api.github.com')` with the combination of https://github.com/nock/nock#disabling-requests, because it asserts the pendingMocks within `isDone()` block